conda_libmamba_solver/solver.py

```diff
@@ -1028,18 +1029,24 @@ def _notify_conda_outdated(
         # manually check base prefix since `PrefixData(...).get("conda", None) is expensive
         # once prefix data is lazy this might be a different situation
         current_conda_prefix_rec = None
+        conda_self_installed = False
+        conda_frozen_file_found = os.path.exists(os.path.join(context.conda_prefix, PREFIX_FROZEN_FILE))
```

Comment:
```suggestion
        conda_frozen_file_found = os.path.exists(os.path.join(context.root_prefix, PREFIX_FROZEN_FILE))
```

I think that'll consider if `ROOT_DIR` is used, and fallback to `context.conda_prefix` (aka `sys.prefix`).